packages/pci-sdk-iframe/src/services/cardData.service.ts

```diff
@@ -64,7 +64,12 @@ function showCardData({ dispatch, cardId, isPCICompliant }: IShowCardDataArgs) {
 		.then((card) => dispatch({ ...card, isLoading: false, message: '', uiStatus: 'CARD_DATA_VISIBLE' }))
 		.catch((err) => {
 			if (_checkIfInvalidAPIKeyError(err)) {
-				return dispatch({ message: 'Invalid API key', uiStatus: 'CARD_DATA_HIDDEN', isLoading: false });
+				return dispatch({
+					message: 'Invalid mobile API key. Refer to the Developer Portal for a valid key.',
+					uiStatus: 'CARD_DATA_HIDDEN',
+					isLoading: false,
+					notificationType: 'negative',
+				});
```

Comment:
Should we open a ticket in **Asana** to write tests verifying these new error messages are shown correctly?